Sources/EdgeNetworkHandlers/NetworkResponseHandler.swift

```diff
@@ -137,8 +137,8 @@ class NetworkResponseHandler {
             let requestEventId = extractRequestEventId(forEventIndex: eventHandle.eventIndex, requestId: requestId)
             handleStoreEventHandle(handle: eventHandle)
 
-            guard let eventHandleAsDictionary = try? eventHandle.asDictionary() else { return }
-            dispatchResponseEvent(handleAsDictionary: eventHandleAsDictionary, requestId: requestId, requestEventId: requestEventId)
+            guard let eventHandleAsDictionary = try? eventHandle.asDictionary() else { continue }
+            dispatchResponseEvent(handleAsDictionary: eventHandleAsDictionary, requestId: requestId, requestEventId: requestEventId, eventSource: eventHandle.type)
```

Comment:
Nit: can we add a comment in here explaining that the handle type is used as event source for response event filtering and it is expected to have a standardized format.